docs/Documentation/Applications/comsol.md

```diff
@@ -37,27 +45,31 @@ Because FastX desktop sessions are supported from DAV nodes shared between multi
 
 However, the performance may be slow and certain display features may behave unexpectedly.
 
-## Running a COMSOL Model in Batch Mode
+## Running a Single-node COMSOL Model in Batch Mode
 You can save your model built in FastX+GUI mode into a file such as `myinputfile.mph`. Once that's available, the following job script shows how to run a single process multithreaded job in batch mode:
 
 ???+ example "Example Submission Script"
 
     ```bash
-    #!/bin/bash
-    #SBATCH --job-name=comsol-batch-1proc
-    #SBATCH --time=00:20:00
-    #SBATCH --nodes=1
-    #SBATCH --account=<your-allocation-id>
-    #SBATCH --output=comsol-%j.out
-    #SBATCH --error=comsol-%j.err
+    #!/bin/bash                                                                                                                                                                                     
+    #SBATCH --job-name="comsol-batch-single-node"                                                                                                                                                   
+    #SBATCH --nodes=1                                                                                                                                                                               
+    #SBATCH --ntasks-per-node=104                                                                                                                                                                   
+    #SBATCH --cpus-per-task=1                                                                                                                                                                       
+    #SBATCH --time=00:10:0        
+    #SBATCH --partition=debug
+    #SBATCH --exclusive
```

Comment:
--exclusive isn't necessary on Kestrel (nodes are always exclusive unless partition is specified as shared). It's not changing any behavior having it in the script, but I think it's best to leave out to avoid any confusion. Same for the other example script.